docs/user-guide/src/bmo/firmware_updates.md

```diff
@@ -0,0 +1,149 @@
+# Firmware Components
+
+Metal3 supports updating firmware and retrieving the current firmware versions
+of the bare metal hosts.
+This feature can be used to update the system firmware (e.g. BIOS) or BMC
+firmware.
+The corresponding functionality in Ironic is called
+[Firmware Updates][1].
```

Comment:
After the rename, the  ironic link does not seem very useful: it focuses on details that are not exposes to Metal3. You could move it to a "See also" section in the end.